src/backer/backering.py

```diff
@@ -473,7 +474,14 @@ def on_post(self, req, rep):
             if self.ledger:
                 try:
                     event = eventing.loadEvent(self.hab.db, pre, serder.saidb)
-                    self.ledger.publishEvent(event)
+
+                    # KEL will push to on-chain icpMessage-controllerSig-witnessSig
+                    # parse witnessSig
+                    data = crt.decode()
```

Comment:
Where is `crt` defined?

Reply:
here
https://github.com/cardano-foundation/cardano-backer/blob/main/src/backer/backering.py#L469
```
rct = self.hab.receipt(serder)
```

Reply:
ok typo then `crt` vs `rct`

Reply:
update, and remove